nvtabular/column_group.py

```diff
@@ -44,28 +54,41 @@ def __init__(self, columns):
         if isinstance(columns, str):
             columns = [columns]
 
+        if not tags:
+            tags = []
+        if isinstance(tags, DefaultTags):
+            tags = tags.value
```

Comment:
Think it would be worth creating a `Taggable` mix-in to handle shared code between `Column` and `ColumnGroup`?